lambda/src/main/scala/pricemigrationengine/model/membershipworkflow/EmailMessage.scala

```diff
@@ -18,9 +18,60 @@ case class EmailPayloadSubscriberAttributes(
     next_payment_date: String,
     payment_frequency: String,
     subscription_id: String,
-    product_type: String
+    product_type: String,
+
+    // SupporterPlus 2024 extension (see comment below)
```

Comment:
why not make these Options?

Reply:
You mean this?

```
 sp2024_contribution_amount: String = ""
 sp2024_previous_combined_amount: String = ""
 sp2024_new_combined_amount: String = ""
```

That's a good question. I tried that, but then wanted to remain backward compatible with existing tests which would have needed to be updated, and I didn't want to do that. Considering that this is code that is going to be thrown away in 13 months, I didn't think of it as a big problem.

Reply:
@tomrf1 
Actually wasn't too bad. https://github.com/guardian/price-migration-engine/pull/1056/commits/f66073fc1997309fb0d1e7960502934ac963fb5b